ui/ui-components/components/export/List.tsx

```diff
@@ -77,18 +77,31 @@ export default function ExportsList(props) {
   }
 
   function getErrorRow(_export: Models.ExportType) {
-    if (!_export.errorMessage) {
-      return null;
-    }
+    if (!_export.errorMessage) return null;
+
+    const delayMessage =
+      ["pending", "processing"].includes(_export.state) &&
+      _export.sendAt > new Date().getTime()
+        ? `Export will be retried after ${formatTimestamp(_export.sendAt)}`
+        : null;
 
     let level = "warning";
-    if (_export.errorLevel === "info") {
+    if (_export.errorLevel === "info" || delayMessage) {
       level = "info";
     }
```

Comment:
I decided to keep the error message the plugin returns along with the retryDelay, but make it less scary looking